### PR TITLE
Closes #7. Change link colours

### LINKED ISSUE #7: Links are difficult to distinguish from regular text (colours are too similar)
The dark blue colour of links in the bootstrap theme is not much different from black, and links are not underlined except on hover, so they can be difficult to spot.

I propose changing the colour of links generally, as I have done for this forked lesson website for a workshop being run now:

https://edcarp.github.io/genomics-r-intro/

Note: The links here have been made a bright blue. I have also changed the colour for links that have already been visited, and that is currently a bright pink, but this change is not necessary, or the colour could be modified to something else.

I can put together a PR for this, but would like to know if it would be preferable for the new CSS to be placed inside of the bootstrap file, or whether you would prefer a separate CSS file (which would also require adding a line to the `base.html`,  and `workshop.html` layout files.

### PR DESCRIPTION
**Pull request by baileythegreen**
...to make them distinguishable from regular text. The colours could be modified. See Issue #7 and examples of how this code should change the appearance here: https://edcarp.github.io/genomics-r-intro/.

@fmichonneau Zhian thought you might have a preference for a different method, coming from a more-informed perspective on how this interacts with the external theme. Let me know if the changes should be done differently.

### CODE REVIEW

**Comment by fmichonneau** [comment]
I'd like to keep the 2 repositories (`styles` and `carpentries-theme`) in sync. I do this by cherry-picking the commits. I'm happy with the changes you propose here but they should be introduced in styles first. It looks like there is already https://github.com/carpentries/styles/pull/511

Could you please submit your PR in the styles repo and then I'll bring the changes here as well? Sorry for the dispersion of the code across all these repos :(

**Comment by baileythegreen** [comment]
@fmichonneau Submit it exactly the same? Easy enough. Should I do it on the branch tied to that pull request, or a new one?

**Comment by fmichonneau** [comment]
I looked quickly but it seems that the same PR would work. Probably a new branch in the styles repo would keep things cleaner. Thanks!

File: assets/css/lesson.scss [hunk 1]
```diff
@@ -225,6 +225,20 @@ a code {
...
+}
+
+a:active,
+a:hover {
+  outline: 0;
+  text-decoration: underline;
+}
+
+a:visited {
+  color: #BC19BD;
```
Comment (baileythegreen) [inline assets/css/lesson.scss:239]:
```suggestion
  color: #1AA4DA;
```

Under Color Oracle, the two new link colours actually did not appear very different for people with deuteranopia.

**Pull request closed without merging**